tools/debug/debugsymbolviewer.py

```diff
@@ -177,8 +177,8 @@ def print_symbol(stack_addr, search_addr, is_app_symbol):
 		if debug:
 			print('Symbol not found for address: {0} in map file'.format(hex(search_addr)))
 
-# Function to search the search_addr for corrupted heap node present in the Address to Symbol mapping table and print its corresponding symbol
-def print_symbol_heap_corruption(search_addr, is_app_symbol):
+# Function to search the search_addr present in the Address to Symbol mapping table and print its corresponding symbol
+def print_symbol_short(search_addr, is_app_symbol):
```

Comment:
I think `short` is not matched with this function. Could you please check this?

Reply:
Changed to print_symbol_only
If you have a better name in mind, please let me know